### PR TITLE
A new loader (Hourglass) added

### PR DESCRIPTION
**Pull request by farshad-vgr**
The new loader (Hourglass) added to src/loader
Hourglass loader passed all the tests successfully
The index.tsx file updated
The version number increased from 5.3.4 to 5.4.4

### CODE REVIEW

File: src/loader/Hourglass.tsx [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+            dur="3s"
+            repeatCount="indefinite"
+          />
+        </g>
+      </g>
+    </svg>
+  )
+}
+
+// const HourglassRotate =
```
Comment (mhnpd) [inline src/loader/Hourglass.tsx]: Can you remove this comments if not used

Comment (mhnpd) [reply src/loader/Hourglass.tsx]: @farshad-vgr Once removed its ready to merge

**Review by mhnpd — CHANGES_REQUESTED** [review]
Please update as per comment

Also added a documentation as in the code pointer 
https://github.com/mhnpd/react-loader-spinner/blob/docs/docs/components/Radio.mdx

**Comment by farshad-vgr** [comment]
I removed useless comments and created the hourglass.mdx file in a separate pull request

**Review by mhnpd — APPROVED** [review]

**Pull request merged**